internal/controller/cluster_controller.go

```diff
@@ -178,6 +179,9 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		}
 		return ctrl.Result{}, err
 	}
+	if configuration.Current.WatchNamespace == "cnpg-system" && cluster.Namespace != configuration.Current.OperatorNamespace {
```

Comment:
a couple of question on top of my head:
- what happens if the operator is installed in another namespace?
- what happens if I have multiple operators installed each one watching a specific namespace?